Reviewed role and optimized. Use sudoedit instead of nano in the example!

ypid opened the pull request.

ypid committed:
Reviewed role and optimized. Use sudoedit instead of nano in the example!

franklinkim commented:
hi, on which system did this not work? CentOS 7? 

```
- name: Installing package
  action: "{{ ansible_pkg_mgr }} pkg={{ sudo_package }} state=present"
```

I would expect ansible to return `yum` if `ansible_os_family` is RedHat!?

ypid commented:
I am not quite sure why I changed that. I found it quite clever and I don’t think it broke something (running it against Debian Jessie works). I am only using Debian based systems, so no. I reviewed your role and decided to go with https://github.com/picotrading/ansible-sudo. I reverted that part of the PR.

franklinkim commented:
ok, thanks.
